app/src/webkit/java/org/mozilla/focus/web/WebViewProvider.java

```diff
@@ -98,6 +98,7 @@ private static void configureDefaultSettings(Context context, WebSettings settin
     public static void applyAppSettings(Context context, WebSettings settings) {
         // We could consider calling setLoadsImagesAutomatically() here too (This will block images not loaded over the network too)
         settings.setBlockNetworkImage(Settings.getInstance(context).shouldBlockImages());
+        settings.setLoadsImagesAutomatically(!Settings.getInstance(context).shouldBlockImages());
```

Comment:
loadsImagesAutomatically by shouldBlockImages?

Reply:
LoadsImagesAutomatically this method controls loading of all images, including those embedded using the data URI scheme. But our spec don't specify these.

Reply:
Oh sorry, I didn't see that !. Sorry!!!